templates/managed_services/mas_debezium/consumer_pod.j2

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Any reason to use template and not python to create the pod?

Reply:
Yes, due to the issue I was facing with passing the kcat consuming command via `pod.execute` after creating the pod as I mentioned in [this](https://github.com/RedHatQE/managed-services-integration-tests/pull/50#discussion_r1122894501) comment.
Might be a better way to get this done, let me know what you think.

Reply:
Are you going to remove this and use python code to create the pod?

Reply:
yes

Reply:
done, let me know if its ok